integrations/nlu-snips/src/SnipsNlu.ts

```diff
@@ -79,9 +83,9 @@ export class SnipsNlu extends NluPlugin<SnipsNluConfig> {
     const locale: string = this.getLocale(jovo.$request);
 
     const model: JovoModelData =
-      this.config.models?.[locale] ||
-      (this.config.modelsDirectory
-        ? require(resolve(joinPaths(this.config.modelsDirectory, locale)))
+      this.config.dynamicEntities?.models?.[locale] ||
+      (this.config.dynamicEntities?.modelsDirectory
+        ? require(resolve(joinPaths(this.config.dynamicEntities.modelsDirectory, locale)))
         : {});
 
     if (!model.inputTypes) {
```

Comment:
Could you change this to reflect the changes to the model? `entityTypes`, `entities`

Reply:
Done :+1: